pkg/rancher-desktop/components/Nav.vue

```diff
@@ -172,4 +181,19 @@ a {
   font-size: 0.75rem;
 }
 
+@media (prefers-color-scheme: dark) {
```

Comment:
Since this part is very non-obvious, there should be a comment saying what this is for. Ideally a short blurb on how it works too.